Dockerfile

```diff
@@ -12,4 +12,6 @@ COPY . /code
 
 RUN pip install --no-cache pipenv==2020.11.15
 
+RUN pipenv install --system --deploy
```

Comment:
remove the same from start.sh, there'll be a redundant installation of dependencies